source/numerics.tex

```diff
@@ -51,7 +51,7 @@
 These include arithmetic types,
 pointers, the library class
 \tcode{complex},
```

Comment:
While we are here...should we also change this to say "specializations of `complex`" or something similar? `complex` is certainly not a class.

Reply:
Akin? Analogous to? 

"Comparable to" would probably be the most natural term, but that's overloaded in C++ too.